docs/secpoll.zone

```diff
@@ -383,3 +383,4 @@ dnsdist-1.5.0-rc3.security-status                          60 IN TXT "2 Unsuppor
 dnsdist-1.5.0-rc4.security-status                          60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
 dnsdist-1.5.0.security-status                              60 IN TXT "1 OK"
 dnsdist-1.5.1.security-status                              60 IN TXT "1 OK"
+dnsdist-1.6.0-alpha1.security-status                       60 IN TXT "2 Unsupported pre-release (no known vulnerabilities)"
```

Comment:
I believe this should say '1 OK' if there's nothing newer.

Reply:
I always wonder about that, does "1 OK" mean that the release is supported?

Reply:
We could also do '1 Unsupported pre-release', I suppose. I think we'd support the alpha1 until anything newer comes out though.

Reply:
In my mind the use of "2" in that particular case was a way of signalling that this release is not a stable one, but I'll move to "1 OK".

Reply:
The problem, IIRC, is that 2 will cause alerts in people's monitoring.